packages/material-ui/package.json

```diff
@@ -53,6 +53,7 @@
     "normalize-scroll-left": "^0.2.0",
     "popper.js": "^1.14.1",
     "prop-types": "^15.7.2",
+    "react-is": "^16.12.0",
```

Comment:
This should be relaxed to `^16.8` which matches our peer dep on `react`

Reply:
Done :)